articles/aks/use-wasi-node-pools.md

```diff
@@ -58,15 +58,15 @@ az provider register --namespace Microsoft.ContainerService
 
 ## Limitations
 
-* Currently, there are only containerd shims available for [spin][spin] and [slight][slight] applications, which use the [wasmtime][wasmtime] runtime. In addition to wasmtime runtime applications, you can also run containers on WASM/WASI node pools.
-* You can run containers and wasm modules on the same node, but you can't run containers and wasm modules on the same pod.
-* The WASM/WASI node pools can't be used for system node pool.
-* The *os-type* for WASM/WASI node pools must be Linux.
-* You can't use the Azure portal to create WASM/WASI node pools.
+* Currently, WASI Node Pools only support [Spin][spin] applications which use the [Wasmtime][wasmtime] runtime, and native Linux containers.
```

Comment:
```suggestion
* Currently, WASI Node Pools only support [Spin][spin] applications, which use the [Wasmtime][wasmtime] runtime and native Linux containers.
```

I'm sorry. This was from my feedback. I think you do you need the comma in front of which.